cypress/features/regression/preview.feature

```diff
@@ -42,7 +42,7 @@ Feature: Preview default component
 
   @positive
   Scenario Outline: Set lines to <lines>
-    When I open default "Preview" component in noIFrame with "preview" json from "commonComponents" using "<nameOfObject>" object name
+    When I open basic "Preview Test" component in noIFrame with "preview" json from "commonComponents" using "<nameOfObject>" object name
```

Comment:
this TC (don't forget about `.json` file) could be deleted - it is covered by `chromatic`